system/edit-config

```diff
@@ -112,8 +112,10 @@ check_editor() {
 running_in_container() {
   [ -e /.dockerenv ] && return 0
   [ -e /.dockerinit ] && return 0
-  [ -r /proc/1/environ ] && tr '\000' '\n' </proc/1/environ | grep -Eiq '^container=podman' && return 0
+  [ -e /run/.containerenv ] && return 0
+  [ -r /proc/1/environ ] && tr '\000' '\n' </proc/1/environ | grep -Eiq 'container=' && return 0
```

Comment:
I think we can remove this line, but I doubt it work. The `[ -e /run/.containerenv ]` check is correct https://github.com/containers/podman/issues/3586#issuecomment-661918679. Additionally, can check for `container` env var.

Reply:
> Additionally, can check for container env var.

That’s what this line is doing, it’s just doing so by looking at PID 1 instead of the current process, which we need to do because the current process is _not_ guaranteed to have that variable set.